### ISSUE TITLE
Please Consider Adding `files` Field to Package 

### ISSUE THREAD

**Issue by shellscape** [d4rkr00t/whybundled#17]
Hey there! Nifty project, and we're taking a look at using it in a new sub-command for [`webpack-command`](https://github.com/webpack-contrib/webpack-command). However, we noticed that the package is published with _everything_ in the repo:

```console
→ ls /Downloads/package -aln
total 28
drwxr-xr-x   15 501 20   480 Jul  3 22:54 .
drwx------+ 643 501 20 20576 Jul  3 22:54 ..
-rw-r--r--    1 501 20   145 Dec 31  1969 .editorconfig
-rw-r--r--    1 501 20   103 Dec 31  1969 .flowconfig
-rw-r--r--    1 501 20    30 Dec 31  1969 .prettierignore
-rw-r--r--    1 501 20   282 Dec 31  1969 .travis.yml
drwxr-xr-x    3 501 20    96 Jul  3 22:54 .vscode
-rw-r--r--    1 501 20  3821 Dec 31  1969 README.md
drwxr-xr-x    6 501 20   192 Jul  3 22:54 assets
-rwxr-xr-x    1 501 20  1577 Dec 31  1969 cli.js
drwxr-xr-x    5 501 20   160 Jul  3 22:54 commands
drwxr-xr-x   15 501 20   480 Jul  3 22:54 fixtures
drwxr-xr-x    3 501 20    96 Jul  3 22:54 flow-typed
drwxr-xr-x   10 501 20   320 Jul  3 22:54 lib
-rw-r--r--    1 501 20  1791 Dec 31  1969 package.json
```

That makes the package weigh in at a hefty [17.6 MB](https://packagephobia.now.sh/result?p=whybundled) - ouch! Any chance you'd consider adding a `files` field to `package.json` and giving that a publish?

**Comment by d4rkr00t** [comment]
Released as [v1.3.0](https://github.com/d4rkr00t/whybundled/releases/tag/v1.3.0)

**Comment by shellscape** [comment]
Thanks for getting that out so quickly 🎉
